Userland/Libraries/LibGL/SoftwareGLContext.cpp

```diff
@@ -925,6 +925,23 @@ void SoftwareGLContext::gl_blend_func(GLenum src_factor, GLenum dst_factor)
     m_rasterizer.set_options(options);
 }
 
+void SoftwareGLContext::gl_shade_model(GLenum mode)
+{
+    if (m_in_draw_state) {
```

Comment:
Looks like I'm gonna have to go back at some point and insert all the listing calls to these... :cry:

Reply:
Hang on, I'm just gonna add them.